pynucastro/networks/rate_collection.py

```diff
@@ -159,11 +178,41 @@ def eval_abar(self):
             xvec[i] = self.X[n]
         return 1. / np.sum(xvec / avec)
 
-    def __str__(self):
-        ostr = ""
-        for k in self.X:
-            ostr += f"  X({k}) : {self.X[k]}\n"
-        return ostr
+    def bin_as(self, nuclei, *, verbose=False):
```

Comment:
There is a * placeholder in the function. Is there a significant role for it?

Reply:
that means the options that follow are keyword only

https://peps.python.org/pep-3102/

it makes the interfaces a bit easier / less prone to error

Reply:
good idea.  I'll think up an example test for this.